specs/examples/benchmark-dataset/transliteration-benchmark-dataset/data.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "sourceText": "bhaarat-nepaal seema ke kareeb hone ke chalate is pul ka rananeetik mahatv bhee hai",
+        "targetText": "भारत-नेपाल सीमा के करीब होने के चलते इस पुल का रणनीतिक महत्व भी है"
```

Comment:
Can the records be a sentence or just a single word?

Reply:
ideally its words only, editing now.